### PR TITLE
en: update scale tidb-binlog

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1572
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/deploy-tidb-binlog.md [hunk 1]
```diff
@@ -396,6 +376,9 @@ The steps are as follows:
         ```shell
         kubectl exec binlogctl -n ${namespace} -- /binlogctl -pd-urls=https://${cluster_name}-pd:2379 -cmd update-pump -node-id ${cluster_name}-pump-${ordinal_id}:8250 --state offline -ssl-ca "/etc/binlog-tls/ca.crt" -ssl-cert "/etc/binlog-tls/tls.crt" -ssl-key "/etc/binlog-tls/tls.key"
         ```
+> **Note:**
+>
+> Do not scale in Pump nodes to 0. Otherwise, [Pump nodes are removed completely](#remove-pump-nodes-completely).
```
Comment (qiancai) [inline en/deploy-tidb-binlog.md:382]: Perhaps we could place this note in step 1 to avoid users setting ${pump_replicas} to 0.

**Review by qiancai — COMMENTED** [review]
@ran-huang. some minor comments, PTAL

**Comment by ran-huang** [comment]
@qiancai Comments addressed. PTAL again~

**Review by qiancai — APPROVED** [review]
LGTM

**Review by better0332 — APPROVED** [review]

**Pull request merged**